engine/worldmap.py

```diff
@@ -20,8 +20,8 @@ def SetMapTile(self, x, y, tile):
     def GetLocationDescription(self, x, y):
         return self.map[x][y].toString()
 
-    def AddItemAt(self, x, y, item_description):
-        self.map[x][y].add_item(Detritus(item_description.lower()))
+    def AddItemAt(self, x, y, item_description, item_synonyms=[], item_obtainable=False):
```

Comment:
Now that A Detritus can take in several inputs in its constructor, it makes more sense to change AddItemAt() to take in a Detritus Item instead of all the same parameters as the constructor. Let's refactor this to be
AddItemAt(self, x, y, item)  or even better AddItemAt(self, point, item)